cartridge/lua-api/failover.lua

```diff
@@ -170,27 +171,30 @@ local function promote(replicaset_leaders, opts)
         return nil, PromoteLeaderError:new('There is no active coordinator')
     end
 
-    local _, err = rpc.call(
+    local ok, err = rpc.call(
```

Comment:
Don't understand why this change is necessary

Reply:
It isn't necessary. I can remove it if you insist.

Reply:
It depends on the way how errors are handled in cartridge. I'd prefer to use `err ~= nil` but if `ok == nil` is preferable for the cartridge, then you should leave this as is.

Reply:
I think checking `ret == nil` is more suitable in general. The only exception are methods that can return `nil` without an error, but it's not the case

Reply:
So I'll leave it as is.